language_service/src/state/tests.rs

```diff
@@ -711,6 +711,7 @@ fn notebook_document_lints() {
                                 kind: Ast(
                                     RedundantSemicolons,
                                 ),
+                                code_action_edits: [],
```

Comment:
Shouldn't these older lints now be updated to return code action edits as well? It's confusing that some of the quickfix implementations are defined along with the lints, and some of them are inline in the language service.

Reply:
We'd need to plumb through the Compilation through to the AST lints, as we did with the HIR lints. @orpuente-MS and I had discussed this and wanted to avoid doing that until necessary. I agree that the current way where some code actions come from two different sources is confusing though, so I'd be fine to do that work. Do you think that that would be worth it?

Reply:
Actually, strike that. Since these code actions only remove characters, I don't think we'll need to query the source for anything, meaning we don't need the compilation.

Reply:
Okay I've wired up the AST lints to also use the `code_action_edits` field.